src/spotted/data/user.py

```diff
@@ -49,6 +62,14 @@ def banned_users(cls) -> "list[User]":
             for row in DbManager.select_from(table_name="banned_users", select="user_id, ban_date")
         ]
 
+    @classmethod
+    def muted_users(cls) -> "list[User]":
```

Comment:
why are we using the double quotes? `list[User]` does not work properly?

Reply:
Since this is a method of the User class and needs a forward reference. The double quotes are required

Reply:
Since this i a method of the class, User is not yet defined, and needs a forward reference. The quotes are mandatory

Reply:
I couldn't explain this any way better than TendTo